### PR TITLE
:lock: (kktix) store kktix api into Variable

### PR DESCRIPTION
**Pull request by david30907d**
## Types of changes

- **Other (please describe)**

## Description

hide KKTIX's url path to keep @gtbcoding 's secret

## Expected behavior

https://discord.com/channels/752904426057892052/900721883383758879/1016745608935120956

### CODE REVIEW

File: dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py [hunk 1]
```diff
@@ -71,8 +72,7 @@ def get_event_metadatas(condition_filter: Callable) -> List[Dict]:
     Fetch all the ongoing events
     """
     event_list_resp = HTTP_HOOK.run_with_advanced_retry(
-        endpoint="/api/v2/hosting_events?only_not_ended_event=true",
-        _retry_args=RETRY_ARGS,
+        endpoint=Variable.get("kktix_events_endpoint"), _retry_args=RETRY_ARGS,
```
Comment (david30907d) [inline dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py:75]: hide url path as well

File: dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def _get_attendance_book_id(event_id: int) -> int:
     Fetch attendance books
     """
     attendance_books_resp = HTTP_HOOK.run_with_advanced_retry(
-        endpoint=f"/api/v2/hosting_events/{event_id}/attendance_books",
+        endpoint=f"{Variable.get('kktix_hosting_events_endpoint_prefix')}/{event_id}/attendance_books",
```
Comment (david30907d) [inline dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py:89]: ditto

File: README.md [hunk 3]
```diff
@@ -126,6 +126,7 @@ Please use Gitlab Flow, otherwise you cannot pass dockerhub CI
             * KLAVIYO_KEY: Create from https://www.klaviyo.com/account#api-keys-tab
             * KLAVIYO_LIST_ID: Create from https://www.klaviyo.com/lists
             * KLAVIYO_CAMPAIGN_ID: Create from https://www.klaviyo.com/campaigns
+            * kktix_events_endpoint: url path of kktix's `hosting_events`, ask @gtb for details!
```
Comment (david30907d) [inline README.md:129]: @gtbcoding quick pass?

File: dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py [hunk 4]
```diff
@@ -71,7 +72,7 @@ def get_event_metadatas(condition_filter: Callable) -> List[Dict]:
     Fetch all the ongoing events
     """
     event_list_resp = HTTP_HOOK.run_with_advanced_retry(
-        endpoint="/api/v2/hosting_events?only_not_ended_event=true",
+        endpoint=f"{Variable.get('kktix_events_endpoint')}?only_not_ended_event=true",
```
Comment (david30907d) [inline dags/ods/kktix_ticket_orders/udfs/batch_kktix2mailer.py:75]: ditto, hide url path as well

**Comment by gtbcoding** [comment]
LGTM

**Pull request merged**